mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactPdfView.js

```diff
@@ -2,11 +2,13 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import './ShowArtifactImageView.css';
 import { getSrc } from './ShowArtifactPage';
-import { Document, Page } from 'react-pdf';
+import { Document, Page, pdfjs } from 'react-pdf';
 import { Pagination, Spin } from 'antd';
 import { getArtifactBytesContent } from './ShowArtifactUtils';
 import './ShowArtifactPdfView.css';
 
+pdfjs.GlobalWorkerOptions.workerSrc = `${process.env.PUBLIC_URL}/pdf.worker.js`;
```

Comment:
Can we provide some comments with a link to the documentation of how `workerSrc` should be used?

Reply:
Done!